### PR TITLE
monkeysAudio: 10.76 -> 10.81

### PR DESCRIPTION
**Pull request by jopejoe1**
Also dropped mac package as that is just an old version of monkeysAudio and also updated the license to bsd3 as they changed it.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/mo/monkeysAudio/package.nix [hunk 1]
```diff
@@ -5,15 +5,20 @@
...
     url = "https://monkeysaudio.com/files/MAC_${
       builtins.concatStringsSep "" (lib.strings.splitString "." finalAttrs.version)}_SDK.zip";
-    hash = "sha256-ropQZraOombq6zG5vXU/kBtQggy30ErbU79gbEtrIxs=";
+    hash = "sha256-sI2u+H/ewva9r+g5xSNqal0DMul+a9Y4FV6dEzulvSI=";
     stripRoot = false;
   };
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-std=c++11"
+  ];
```
Comment (doronbehar) [inline pkgs/by-name/mo/monkeysAudio/package.nix:20]: Would be nice to comment here what kind of errors have your experienced that made you put this, and how did you realize this is the way to fix the issue you experienced.

File: pkgs/by-name/mo/monkeysAudio/package.nix [hunk 2]
```diff
@@ -22,10 +27,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "APE codec and decompressor";
     platforms = platforms.linux;
     mainProgram = "mac";
-    # This is not considered a GPL license, but it seems rather free although
-    # it's not standard, see a quote of it:
-    # https://github.com/NixOS/nixpkgs/pull/171682#issuecomment-1120260551
-    license = licenses.free;
+    license = licenses.bsd3;
```
Comment (doronbehar) [inline pkgs/by-name/mo/monkeysAudio/package.nix:30]: So they changed their license? Could you reference the upstream change?

File: pkgs/by-name/ma/mac/package.nix [hunk 3]
```diff
@@ -1,36 +0,0 @@
-{ lib, stdenv, fetchurl, fetchpatch, yasm }:
-
-stdenv.mkDerivation rec {
-  pname = "mac";
-  version = "4.11-u4-b5-s7";
-
-  src = fetchurl {
-    url = "https://www.deb-multimedia.org/pool/main/m/monkeys-audio/monkeys-audio_${version}.orig.tar.gz";
-    sha256 = "16i96cw5r3xbsivjigqp15vv32wa38k86mxq11qx1pzmpryqpqkk";
-  };
```
Comment (doronbehar) [inline pkgs/by-name/ma/mac/package.nix:10]: Thanks for noticing this!

**Review by doronbehar — COMMENTED** [review]
Looks good overall, thanks!

**Comment by doronbehar** [comment]
Will merge when CI is green.

File: pkgs/by-name/mo/monkeysAudio/package.nix (hunk 2 above)
Comment (jopejoe1) [reply pkgs/by-name/mo/monkeysAudio/package.nix:31]: Was changed in version 10.18 see changelog for info https://monkeysaudio.com/versionhistory.html

**Pull request merged**